.github/workflows/deploy-distributed-press.yaml

```
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Check out the repository
+      - uses: actions/checkout@v3
+
+      # Step 2: Set up Node.js
+      - name: Set up Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      # Step 3: Install dependencies and generate static files
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Generate site files
+        run: node ./src/index.js
```

```suggestion
        run: node ./
```

Add a `main` field in the package.json